minion/server/core.py

```diff
@@ -160,3 +160,14 @@ def process(job, config, keep_data):
 
         if not keep_data:
             shutil.rmtree(local_repo_path)
+
+def stop_server():
```

Comment:
Never worked with multiprocessing before. That is why I overlooked it. Will read the docs and try to find a solution, thanks!